test/functional/test-resources.js

```diff
@@ -1160,6 +1114,27 @@ describe('Resources changeSize', () => {
       expect(resources.relayoutTop_).to.equal(resource1.layoutBox_.top);
     });
 
+    it('in vp should NOT call overflowCallback if new height smaller', () => {
+      resources.scheduleChangeSize_(resource1, 10, 11, false);
+      resources.mutateWork_();
+      expect(resources.requestsChangeSize_.length).to.equal(0);
+      expect(resource1.changeSize.callCount).to.equal(0);
+      expect(overflowCallbackSpy.callCount).to.equal(0);
+    });
+
+    it('in viewport should NOT change size and calls overflowCallback', () => {
+      resources.scheduleChangeSize_(resource1, 111, 222, false);
+      resources.mutateWork_();
+      expect(resources.requestsChangeSize_.length).to.equal(0);
+      expect(resource1.changeSize.callCount).to.equal(0);
+      expect(overflowCallbackSpy.callCount).to.equal(1);
+      expect(overflowCallbackSpy.firstCall.args[0]).to.equal(true);
+      expect(overflowCallbackSpy.firstCall.args[1]).to.equal(111);
+      expect(overflowCallbackSpy.firstCall.args[2]).to.equal(222);
```

Comment:
you can merge the 3 lines to
`expect(overflowCallbackSpy.firstCall.args).to.jsonEqual([true, 111, 222]);`